dokan-class.php

```diff
@@ -312,6 +312,7 @@ public function init_classes() {
 
         if ( is_admin() ) {
             $admin_services = $this->get_container()->get( 'admin-service' );
+            $admin_dashboard_services = $this->get_container()->get( 'admin-dashboard-service' );
```

Comment:
- Pls sync with the develop
- Implement the `Hookable` interface to the service classes.

[See the `Hookable`  resolution](https://github.com/getdokan/dokan/blob/a169e59f4af7870e4e433bae212a81c4757eec2c/dokan-class.php#L273-L276)

Reply:
Implemented